core/src/main/java/org/apache/accumulo/core/spi/compaction/CompactionDirectives.java

```diff
@@ -30,19 +30,4 @@
    * @return The service where a compaction should run.
    */
   CompactionServiceId getService();
-
-  /**
-   * @since 2.1.0
-   */
-  public static interface Builder {
```

Comment:
This builder was intended to be used by someone writing a CompactionDispatcher.  If someone were to write a compaction dispatcher now they would need to implement CompactionDirective.

Reply:
If CompactionDirective is for returning information to the user, than it shouldn't need a builder (similar to how we return Environment objects). If the user needs to set objects in it for their implementation, perhaps a separate utility method would be better.